frontend/include/chpl/resolution/resolution-queries.h

```diff
@@ -558,6 +558,15 @@ CallResolutionResult resolveTheseCall(ResolutionContext* rc,
                                       const types::QualifiedType& followThis,
                                       const CallScopeInfo& inScopes);
 
+/**
+  Given a type's symbol path and the name of a default function, return a
+  BuilderResult storing the generated default function uAST.
+*/
```

Comment:
It is interesting. At first I would think that this function should return a `Function*` ? I see that it returns a `BuilderResult*` to integrate into `parseFileContainingIdToBuilderResult`. IMO it might be worth renaming the function to emphasize this -- but at least it would be good to put a comment explaining that this is intended to be called by parsing queries that need to handle a `BuilderResult*` for generality.

Reply:
Renamed to ``builderResultForDefaultFunction``.